JUnit version 4.13.2
.....E.............................E.....E................
Time: 1.535
There were 3 failures:
1) test34(com.google.common.reflect.ClassPath_ESTest)
java.lang.AssertionError: expected:<1> but was:<11>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at com.google.common.reflect.ClassPath_ESTest.test34(ClassPath_ESTest.java:473)
2) test51(com.google.common.reflect.ClassPath_ESTest)
java.lang.AssertionError: Expecting exception: NoSuchElementException
	at org.junit.Assert.fail(Assert.java:89)
	at com.google.common.reflect.ClassPath_ESTest.test51(ClassPath_ESTest.java:649)
3) test09(com.google.common.reflect.ClassPath_ESTest)
java.lang.AssertionError: Exception was not thrown in java.io.File but in com.google.common.base.Preconditions.checkArgument(Preconditions.java:131): org.evosuite.runtime.mock.java.lang.MockThrowable
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:111)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:48)
	at com.google.common.reflect.ClassPath_ESTest.test09(ClassPath_ESTest.java:148)

FAILURES!!!
Tests run: 55,  Failures: 3

